lib/validateCreateArgs.js

```diff
@@ -44,6 +44,10 @@ function validateCreateArgs(args) {
 
     }
 
+    if (args.profile.securityVersion === "v1" && args.secure){
```

Comment:
I can't remember if we had decided to support custom issuers and custom audiences in the first iteration. What would you think about logic whereby either both `--jwt-audience` and `--jwt-issuer` are required or neither must be present. If neither are present, then we would fall back to discovering the defaults from the cluster.

Reply:
@ggoodman do you mean additional CLI flags with values that take precedence over the call to the discovery endpoint?  I think it is a good idea and I can add it to the CLI.